app/views/drawless_groups/show.html.erb

```diff
@@ -1,20 +1,20 @@
 <%= render partial: 'groups/group_details' %>
 <% if policy(DrawlessGroup.new(@group)).select_suite? %>
-  <h2><%= suite_str(@group) %></h2>
+  <h3><%= suite_str(@group) %></h3>
   <% if @group.suite %>
```

Comment:
Yea, it ends up being the prompt to select a suite, which is restricted to admins. We can add suite display as an alternative - this view def needs some help.